Donation widget/quick option

duranmla opened the pull request:
**What:**
Allow user to quick select predefined amount values

**Why:**
relates to Development of Donation widget for upcoming release

**How:**
- Add new `QuickOption` component within the donation widget
- Integrate new component into Donation Once and Monthly forms

> Disclaimer: it may be some complexity that we can improve/remove over the next iterations of the widget. Nevertheless, bare in mind that the layout of `QuickOption` has
been done trying to facilitate the iteration over the options without breaking the way all the options are place.

**Media:**
![image](https://user-images.githubusercontent.com/1425162/93666671-4f373f00-fa80-11ea-8736-8007ba5caa62.png)

https://www.loom.com/share/ce0fd7ac65ab4176baa4d514e53c936f

orlando commented on tailwind.config.js:
```diff
@@ -54,6 +54,7 @@ module.exports = {
         sans: ['Roboto', 'Helvetica', 'Arial', 'sans-serif']
       },
       fontSize: {
+        tiny: '0.6rem',
```
should we keep the same pattern as the others? xxs for example

orlando reviewed (commented):
just a small comment

orlando commented:
Also, if we can limit the amount to 50k that would be great, just like this
![image](https://user-images.githubusercontent.com/849872/93668411-ac67cc00-fa51-11ea-8498-9bf7166e07c4.png)

duranmla commented:
Do we have some link for the contact us? I wonder if it is convenient to just set a maximum on the input number or we actually want something else.

duranmla commented:
I've addressed the feedback @orlando 
![image](https://user-images.githubusercontent.com/1425162/93688612-4f0e6200-fac7-11ea-8082-8459e4460648.png)

_The call to action appears while typing no on submit as it was easy for me to implement and IMO is better to alert the user as soon as possible._

orlando commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/debtcollective/home/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
